recipes/mtsv/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version="1.0.1" %}
+{% set sha256="de98f8470c691eb445a948ed6ac9d44d2183e4257820e5315916497eb13c7b82" %}
+
+package:
+  name: 'mtsv'
+  version: {{ version }}
+
+source:
+  url: https://github.com/FofanovLab/MTSv/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [osx]
+  entry_points:
+    - mtsv = mtsv.main:main
+    - mtsv_setup = mtsv.mtsv_prep.main:main
+    - mtsv_plugin = mtsv.mtsv_plugin.main:main
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - rust >=1.18.0
+  host:
+    - zlib
+    - python==3.6.0
```

Comment:
Are you certain you need to pin this strictly? Would `- python >=3.6` not do?

Reply:
Changed pinning